webapp/src/views/projects/translations/TranslationTools/TranslationMachine.tsx

```diff
@@ -0,0 +1,76 @@
+import { makeStyles } from '@material-ui/core';
+import { components } from 'tg.service/apiSchema.generated';
+import { getProviderImg } from './getProviderImg';
+import { useTranslationTools } from './useTranslationTools';
+
+type SuggestResultModel = components['schemas']['SuggestResultModel'];
+
+const useStyles = makeStyles((theme) => ({
+  container: {
+    display: 'flex',
+    flexDirection: 'column',
+  },
+  item: {
+    padding: theme.spacing(1, 1.25),
+    display: 'grid',
+    gap: theme.spacing(0, 1),
+    gridTemplateColumns: '20px 1fr',
+    cursor: 'pointer',
+    transition: 'all 0.1s ease-in-out',
+    transitionProperty: 'background color',
+    '&:hover': {
+      background: theme.palette.extraLightBackground.main,
+      color: theme.palette.primary.main,
+    },
+  },
+  source: {
+    marginTop: 3,
+  },
+  value: {
+    fontSize: 15,
+    alignSelf: 'center',
+  },
+}));
+
+type Props = {
+  data: SuggestResultModel | undefined;
+  operationsRef: ReturnType<typeof useTranslationTools>['operationsRef'];
+};
+
+export const TranslationMachine: React.FC<Props> = ({
```

Comment:
TranslationMachine is kind of inconvenient name. MachineTraslations is how we call it.